htmd/ffevaluation/ffevaluate.py

```diff
@@ -34,6 +34,8 @@ def run(self, coords, box=None):
             box = np.zeros((3, coords.shape[2]), dtype=np.float32)
         # from IPython.core.debugger import set_trace
         # set_trace()
+        assert box.shape[0] == 3 and box.shape[1] == coords.shape[2], 'Box dimensions have to be (3, numFrames), your Molecule has box of shape {}'.format(box.shape)
+
```

Comment:
This should throw `ValueError` exception, rather than assert. As the error can be triggered by a user directly passing a wrong `box`.